README.md

```diff
@@ -172,3 +136,37 @@ Run the following from the top level concord-bft directory:
 You can use the simpleTest.py script to run various configurations via a simple
 command line interface.
 Please find more information [here](./tests/simpleTest/README.md)
+
+## Directory Structure
+
+
+- [bftengine](./bftengine): concord-bft codebase
+	- [include](./bftengine/include): external interfaces of concord-bft (to be used by client applications)
+	- [src](./bftengine/src): internal implementation of concord-bft
+    - [tests](./bftengine/tests): tests and usage examples
+- [threshsign](./threshsign): crypto library that supports digital threshold signatures
+	- [include](./threshsign/include): external interfaces of threshsign (to be used by client applications)
+	- [src](./threshsign/src): internal implementation of threshsign
+    - [tests](./threshsign/tests): tests and usage examples
+- [scripts](./scripts): build scripts
+- [tests](./tests): BFT engine system tests
+
+## Contributing
+
+
+The concord-bft project team welcomes contributions from the community. If you wish to contribute code and you have not
+signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any
+questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq). For more detailed information,
+refer to [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Community
+
+
+[Concord-BFT Slack](https://concordbft.slack.com/).
+
+Get Slack invitation via this [link](https://join.slack.com/t/concordbft/shared_invite/enQtNTAyMzg0MTk0NzU0LWViMjA4ODQ4OTcxYmVkMWU4NGYyNTZmZDQ0ZTIzZTNiZmIyZmI5Mjg0NDZjNjBkYmI2Yjg1Mzg1NGUyNjQ0OWQ) or send request to <concordbft@gmail.com>.
```

Comment:
This link doesn't seem to work anymore. Invitation links usually expire - we should provide a more persistent entry point here.

Reply:
@salieri11 , what would you suggest?